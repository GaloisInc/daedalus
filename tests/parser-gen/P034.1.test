run
P034.ddl
--gen
--input=inputs/P034.1.inp
