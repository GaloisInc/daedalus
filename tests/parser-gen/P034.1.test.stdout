[WARNING] ./P034.ddl:10:9--13:5:
            Using unbiased choice may be inefficient.
--- Found 2 results:
{ a: 'a'
, t: {x : "a"}
}
 
{ a: 'a'
, t: {y : "b"}
}
